css/main.css

```diff
@@ -96,10 +96,12 @@ main {
     text-align: right;
     padding: 0 0.5rem;
 }
-.station-info .entry--logo {
+
+.entry .logo {
```

Comment:
Here it would make sens to keep on using `.entry--logo` since you designate a child of an `.entry` block.
`.station-info .entry--logo` was not good but `.entry .entry--logo` would be ok.